.github/actions/publish/action.yml

```diff
@@ -59,4 +59,4 @@ runs:
       shell: bash
       run: cd protocol-models/typescript && npm publish
       env:
-        NODE_AUTH_TOKEN: ${{ inputs.npm_token }}
+        NPM_TOKEN: ${{ inputs.npm_token }}
```

Comment:
note: this is per https://docs.npmjs.com/using-private-packages-in-a-ci-cd-workflow#set-the-token-as-an-environment-variable-on-the-cicd-server